release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -172,6 +172,14 @@ With this update, you can now choose from a list of available versions for an Op
 
 //link to content in Operator book TBD
 
+[id="console-supports-aws-sts-detection"]
+===== OperatorHub support for AWS STS mode
+
+With this release, OperatorHub detects when a cluster is running on Amazon Web Services (STS) in Security Token Service (STS) mode. When detected, a "Cluster in STS Mode" message displays to provide additional instructions to ensure the Operator runs correctly. The *Operator Installation* page is also modified in this mode to add the required *role ARN* field.
```

Comment:
I know they called it a mode in the GUI but would avoid talking about it as a mode outside of that context

```suggestion
With this release, OperatorHub detects when a cluster that is running on Amazon Web Services (AWS) is using the Security Token Service (STS). When detected, a "Cluster in STS Mode" message displays to provide additional instructions to ensure the Operator runs correctly. The *Operator Installation* page is also modified in this mode to add the required *role ARN* field.
```

Or, shorter:

```suggestion
With this release, OperatorHub detects when an Amazon Web Services (AWS) cluster is using the Security Token Service (STS). When detected, a "Cluster in STS Mode" message displays to provide additional instructions to ensure the Operator runs correctly. The *Operator Installation* page is also modified in this mode to add the required *role ARN* field.
```